server/src/domain/job/job.service.ts

```diff
@@ -155,8 +156,8 @@ export class JobService {
 
         try {
           const handler = jobHandlers[name];
-          const success = await handler(data);
-          if (success) {
+          const status = await handler(data);
+          if (status === JobStatus.SUCCESS || status == JobStatus.SKIPPED) {
```

Comment:
Did you change any `false` => `SKIPPED`?

Reply:
I most certainly did, ugh.

Reply:
Did you change any `true` => `SKIPPED`? Otherwise just remove skipped so we have the same behavior for now.

Reply:
I made sure that there aren't any `false`  -> `SKIPPED` among the jobs that cause something in onDone. So I'd argue that's fine as is now